css/style.css

```diff
@@ -1,6 +1,163 @@
 /* General page styles */
 
-/* We should use a mobile first approach having out breack point at 768px */
+/* We should use a mobile first approach having out brake point at 768px */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.container {
+	position: relative;
+	margin: auto;
+	max-width: 960px;
+	height: 100%;
+}
+
+.main-nav {
+	position: fixed;
+	top: 0;
+	z-index: 2;
+	width: 100%;
+	height: 75px;
+	overflow-x: hidden;
+	transition: 0.5s;
+}
+
+.colored__main-nav {
+	background-color: rgba(0, 60, 143, 0.8);
+}
+
+.transparent__main-nav {
+	background-color: rgba(0, 60, 143, 0);
+}
+
+.main-nav__logo {
+	position: absolute;
+	top: 11px;
+	left: 20px;
+}
+
+.main-nav__logo img {
+	width: 89px;
+}
+
+.main-nav__burguer {
+	position: absolute;
+	top: 20px;
+	right: 20px;
+	cursor: pointer;
+}
+
+.main-nav__burguer div {
+	width: 40px;
+	height: 7px;
+	background-color: white;
+	transition: 0.5s;
+}
+
+.burguer__bar-1,
+.burguer__bar-2 {
+	margin-bottom: 7px;
+}
+
+.burguer-animation .burguer__bar-1 {
+	transform: rotate(-45deg) translate(0px, 14px);
+	transform-origin: center center;
+	margin: 0;
+}
+
+.burguer-animation .burguer__bar-2 {
+	opacity: 0;
+}
+
+.burguer-animation .burguer__bar-3 {
+	transform: rotate(45deg) translate(0px, -14px);
+	transform-origin: center center;
+}
+
+.main-nav__menu-items {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	top: 0;
+	z-index: -1;
+	width: 100%;
+	height: 0px;
+	margin: 0px;
+	padding: 0px;
+	opacity: 0;
```

Comment:
by seting 
`height: 0px;`
`margin: 0px;`
`padding: 0px;`
`opacity: 0;`

maybe your are looking for `display: none;`

Reply:
It is working as a flex container, if I use the `display:none` instead of `display:flex` it breaks up. It also helps with the further animation. What do you think about it?

Reply:
Discussed offline, this approach is taken to ensure proper animation.

Reply:
Fixed.